src/Makefile.am

```diff
@@ -43,6 +43,8 @@ AM_CPPFLAGS = -I$(top_srcdir)/include        \
 
 bin_PROGRAMS = curl
 
+SUBDIRS = ../docs
```

Comment:
I did not find a reference stating if SUBDIRS works for external directories. If it does, great.